templates/user/participation/more.html.twig

```diff
@@ -0,0 +1,84 @@
+                            <div class="btn text-info p-0"><i class="fa fa-edit"></i> Edit Participation</div>
+                        </a>
+                        <a href="{{ path('user_participations_cancel', { id: participation.id }) }}">
+                            <div class="btn participation-cancel p-0"><i class="fa fa-times-circle"></i> Cancel Participation</div>
+                        </a>
+                    </div>
+                </div>
+            {% endfor %}
+        </div>
+    {% endif %}
+
+    {% if type == 'pendingParticipations' %}
+        <div class="row mb-5">
+            <h2 class="mx-0 mb-5">Participations still waiting for approval</h2>
+            {% for participation in participations %}
+                <div class="user-items-list">
+                    <div class="row w-100">
+                        <a href="{{ path('conferences_show', {slug: participation.conference.slug}) }}">
+                            {{ participation.conference.name }}
+                        </a>, starts on {{ participation.conference.startAt|date('m/d/Y') }}
+                    </div>
+
+                    <div class="row w-100">
+                        <a href="{{ path('user_participations') }}" class="pr-5">
+                            <div class="btn text-info p-0"><i class="fa fa-edit"></i> Edit Participation</div>
+                        </a>
+                        <a href="{{ path('user_participations_cancel', { id: participation.id }) }}">
+                            <div class="btn participation-cancel p-0"><i class="fa fa-times-circle"></i> Cancel Participation</div>
+                        </a>
+                    </div>
+                </div>
+            {% endfor %}
+        </div>
+    {% endif %}
+
+    {% if type == 'rejectedParticipations' %}
+        <div class="row mb-5">
+            <h2 class="mx-0 mb-5">Rejected participations</h2>
+            {% for participation in participations %}
+                <div class="row w-100">
+                    <a href="{{ path('conferences_show', {slug: participation.conference.slug}) }}">
+                        {{ participation.conference.name }}
+                    </a>, starts on {{ participation.conference.startAt|date('m/d/Y') }}
+                </div>
+            {% endfor %}
+        </div>
+    {% endif %}
+
+    {% if type == 'pastParticipations' %}
+        <div class="row mb-5">
+            <h2 class="mx-0 mb-5">Conferences you went to</h2>
+            {% for participation in participations %}
+                <div class="row w-100">
+                    <a href="{{ path('conferences_show', {slug: participation.conference.slug}) }}">
+                        {{ participation.conference.name }}
+                    </a>, starts on {{ participation.conference.startAt|date('m/d/Y') }}
+                </div>
+            {% endfor %}
+        </div>
+    {% endif %}
+{% endblock %}
```

Comment:
Split this template in 4 templates, and make one action per "type" (which is more a filter than a type)